docs/how_to_use/how_to_build.md

```diff
@@ -266,3 +266,17 @@ unzip -o bazel-bin/docs/doxygen/tachyon_api_docs.zip &&
 google-chrome bazel-bin/docs/doxygen/html/index.html
 # generate HTML files and open on Chrome browser.
 ```
+
+## Performance Tuning
+
+### Use Intel OpenMP Runtime Library(libiomp)
+
+By default, Tachyon uses GNU OpenMP (GNU `libgomp`) for parallel computation. On Intel platforms, Intel OpenMP Runtime Library (`libiomp`) provides OpenMP API specification support. It sometimes brings more performance benefits compared to `libgomp`.
```

Comment:
Can you add the pytorch reference here for better credibility? https://pytorch.org/tutorials/recipes/recipes/tuning_guide.html#intel-openmp-runtime-library-libiomp

Reply:
Why do we need this?